spec/acceptance/pe_status_check_spec.rb

```diff
@@ -200,9 +200,22 @@ class {'pe_status_check':
       end
       it 'if S0022 conditions for false are met' do
         run_shell('mv /etc/puppetlabs/license.key /tmp/license.key')
-        run_shell('echo -e "#######################\n#  Begin License File #\n#######################\n \n# PUPPET ENTERPRISE LICENSE - test
-          \nuuid: test\n \nto: test\n \nnodes: 100\n \nlicense_type: Subscription\n \nsupport_type: PE Premium\n \nstart: 2022-01-01\n \nend: 2022-03-29
-          \n#####################\n#  End License File #\n#####################" > /etc/puppetlabs/license.key')
+        myexpiredlicensefile = <<-EOF
```

Comment:
Let's use a `<<~` to allow for indentation. 

```
        myexpiredlicensefile = <<~EOF
          #######################
          #  Begin License File #
          #######################
          # PUPPET ENTERPRISE LICENSE - Puppet Labs
          to: test
          nodes: 100
          license_type: Subscription
          support_type: PE Premium
          start: 2014-02-10
          end: 2022-03-13
          #####################
          #  End License File #
          #####################
          EOF
```

https://ruby-doc.org/core-2.7.2/doc/syntax/literals_rdoc.html